API.md

```diff
@@ -411,6 +423,7 @@ Form components generally all support the following in the `options` object:
     - `required` - boolean specifying whether the data item is required, defaults to `true`.
       This value is ignored for certain form components that are always required, such as radio buttons.
     - `classes` - string to override the default CSS classes used on the form components.
+    - `isPageHeading` - boolean specifying whether the component's label should be used as the page heading. Useful for pages with a single component, defaults to `false`.
```

Comment:
This isn't necessary - the plugin will do this automatically if the page only contains a single form component and that is the first component on the page. I had thought that this was explained in the documentation but if it isn't then it would be useful to add that explanation.